ONSdigital/dp-legacy-cache-proxy
# Proxy pass through for Babbage

### What
A proxy handler had been created to handle all HTTP requests made from dp-frontend-router to Babbage and all HTTP responses from Babbage to the router.

### How to review

1. Run tests `make test`
2. In the dp-frontend-router checkout the branch of [this PR](https://github.com/ONSdigital/dp-frontend-router/pull/444)
Set _LEGACY_CACHE_PROXY_ENABLED_ to true by running this on the terminal
`export LEGACY_CACHE_PROXY_ENABLED=true` and run dp-frontend-router using `make debug`
4. Run dp-legacy-cache-proxy `make debug`
5. Run babbage `./run.sh`
6. Run zebedee `./run-reader.sh` (requires content to have been generated as described in the zebedee README.md prior to this)
7. Make a request to a Babbage endpoint
e.g. http://localhost:20000/economy/environmentalaccounts/bulletins/ukenvironmentalaccounts/2013-06-26

8. HTML content should be returned 

### Who can review

Anyone from ONS
